src/drawer/stories/Drawer.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { css } from "emotion";
+import { Meta } from "@storybook/react";
+import {
+  Drawer,
+  TPlacement,
+  useDrawerState,
+  DrawerBackdrop,
+  DrawerCloseButton,
+  DrawerDisclosure,
+} from "../";
+
+export default {
+  title: "Component/Drawer",
+} as Meta;
+
+const backdropStyles = css`
+  opacity: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  transition: opacity 250ms ease-in-out;
+  background-color: rgba(0, 0, 0, 0.2);
+  &[data-enter] {
+    opacity: 1;
+  }
+`;
+
+const drawerContentStyles = css`
+  opacity: 0;
+  padding: 10px;
+  background-color: white;
+  transition: 250ms ease-in-out;
+  &[data-enter] {
+    opacity: 1;
+  }
```

Comment:
Add a transform on `data-enter` so that example looks like a drawer.
Now, it just shows up, doesn't feel like a drawer 😬